backend/WSS/migrations/0006_auto_20201116_1917.py

```diff
@@ -0,0 +1,162 @@
+# Generated by Django 3.1.3 on 2020-11-16 19:17
```

Comment:
Rename file to a meaningful title like `0006_add_folan.py`

Reply:
renaming causes an error. so I didn't rename it